### DISCUSSION TITLE
GraphQL::Subscriptions::ActionCableSubscriptions error on calling trigger

### DISCUSSION THREAD

**Discussion by xrexonx** [rmosolgo/graphql-ruby#3933] [Q&A] [answered]
Hello, Good Day
Currently  still using graphql-1.12.18 (.define styled implementation) and Im getting this Argument error:

`
Can't reinstall subscriptions. #<GraphQL::Schema:0x00007fb97f10bc70> is using #<GraphQL::Subscriptions::ActionCableSubscriptions:0x00007fb97f109c18>, can't also add GraphQL::Subscriptions::ActionCableSubscriptions
`
Implementation was working fine before. but it's suddenly didn't work and got that argument error.
Tried googling for solution but can't seem to find any solution which leads me to ask question here.

Is someone here have an idea to resolve this?

Thanks,
rex

**Comment by penso** [comment, marked as answer]
I used this to discard the error:

```
   unless YourSchema.subscriptions
     use GraphQL::Subscriptions::ActionCableSubscriptions
   end
```

to only load this once, then found out the error was in my code which made the code reloaded twice.

**Reply by xrexonx** [reply]
Ohh.. Got it.. Thank you @penso
